### PR TITLE
Swift install

### PR DESCRIPTION
**Pull request by jdextercsc**
Support for swift, swift proxy to run on controllers, with storage on some other nodes, specified in the swift group.

### CODE REVIEW

File: roles/common/tasks/main.yml [hunk 1]
```diff
@@ -14,10 +14,25 @@
...
   tags: sysctl
+ 
+- name: deploy resolv.conf
+  template: src=resolv.conf.j2 dest=/etc/resolv.conf
+  tags: dns
 
-- name: attach to subscription manager pool
-  command: subscription-manager attach --pool {{ sm_pool }}
-  tags: sub-mgr
+- name: debug
```
Comment (jcpowermac) [inline roles/common/tasks/main.yml]: Do we need to keep this debug?

Comment (jdextercsc) [inline roles/common/tasks/main.yml]: nope

**Comment by jcpowermac** [comment]
Still reviewing but @jdextercsc you need to rebase.

File: roles/swift-proxy-1/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  tags: swift
+
+
+- name: create the rings
+  command: swift-ring-builder {{ item }} create {{ part_power | default( 16 )}} {{replica_count | default ( 3 ) }} {{min_part_hours | default ( 24 ) }}
+  with_items:
+    - "/etc/swift/object.builder"
+    - "/etc/swift/account.builder"
+    - "/etc/swift/container.builder"
+  tags: swift
```
Comment (jcpowermac) [inline roles/swift-proxy-1/tasks/main.yml]: There is a tab after `swift` that is freaking out the yaml parser.

**Comment by mtnbikenc** [comment]
Looks good :+1:

**Comment by jcpowermac** [comment]
Looks good to me :+1:

**Pull request merged**